test/perf_script.ml

```diff
@@ -74,17 +74,19 @@ let run ?(debug = false) ?ocaml_exception_info ~trace_mode file =
       in
       let should_print_perf_line (event : Event.t) =
         match event with
-        | Ok (Trace event) ->
-          (* Most of a trace is just jumps within a single function. Those are basically
+        | Ok event ->
```

Comment:
Same comment here, we can reduce nesting by matching

```ocaml
| Ok (Power _) | Ok (Sample _) -> false
```

Reply:
Updated